modules/service-ca-certificates.adoc

```diff
@@ -0,0 +1,71 @@
+// Module included in the following assemblies:
+//
+// * authentication/certificate-types-descriptions.adoc
+
+[id="service-ca-certificates_{context}"]
+= Service CA certificates
```

Comment:
Maybe add reference to the service CA expiry of 26 months being longer than the expected upgrade interval for a supported ocp cluster, such that non-control plane consumers of service ca certificates will be refreshed after CA rotation and prior to expiry of the pre-rotation CA?